jsapp/scss/components/_kobo.map.scss

```diff
@@ -166,6 +167,49 @@ body .leaflet-control-layers,
 
   .map-marker-item {
     padding: 4px 6px;
+    position: relative;
+
+    &:after {
```

Comment:
minor cssnazi: `::after` (+ other occurences in this file)

Reply:
I think autoprefixer covers this.

Reply:
It doesn't: https://github.com/postcss/autoprefixer/issues/114 :(

Reply:
Right. I wasn't sure what the difference between `:after` and `::after` was. I've looked it up, and we really don't need to worry about this.